web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -83,13 +90,22 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       ],
     ];
 
+    $dates = $this->getCurrentAndLastMonthDates();
+    $form['date_range_message'] = [
+      '#type' => 'markup',
+      '#prefix' =>  $assetType != 'land' ? '<div class="gfw-hidden">' :'<div class="date-range-text">',
+      '#suffix' => '</div>',
+      '#markup' => $this->t("Showing alerts from <span class='date'>'". $dates['startDate'] ."'</span> to <span class='date'>'" .$dates['currentDate']. "'</span> To see data for more dates please choose a lower level asset (for example CFR)"),
```

Comment:
code warning on line 98 says 'Concatenating translatable strings is not allowed, use placeholders instead and only one string literal'
It should look something like this: 
```
'#markup' => $this->t('Showing alerts from <span class="date">@startdate</span> to <span class="date">@endd  ate</span><br>To see data for more dates please choose a lower level asset (for example CFR)',
  [
    '@startdate' => $dates['startDate'],
    '@enddate' => $dates['currentDate'],
]),
```

Reply:
Could you put the message on two lines? And we don't need the single quotes around the dates in the message

So it would be like 
![image](https://github.com/National-Forestry-Authority/forests/assets/1312689/3bc4dd2e-d754-4a97-99d8-564a54c69d29)

Also it would be good to link from the word CFR to /assets/cfr